tests/test_bandpass.py

```diff
@@ -276,9 +276,26 @@ def test_ne():
     t2 = time.time()
     print 'time for %s = %.2f' % (funcname(), t2-t1)
 
+
+def test_thin():
```

Comment:
Sure.  I added a test_thin() function to test_sed.py that checks that thinning the CWW_E_ext.sed spectrum works as expected.